src/main/kotlin/apply/application/CheaterService.kt

```diff
@@ -14,15 +13,15 @@ class CheaterService(
     private val cheaterRepository: CheaterRepository
 ) {
     fun findAll(): List<CheaterResponse> = cheaterRepository.findAll().map {
-        val applicant = applicantRepository.findByIdOrNull(it.applicantId)!!
+        val applicant = applicantRepository.findByEmail(it.email)!!
```

Comment:
이부분 사실 아론은 예전 코드형식에 맞추신 것 같은데 이 부분만 null 처리가 `!!` 로 되어 있는 것 같아서요. 혹시 다른 것처럼 ?:
```suggestion
        val applicant = applicantRepository.findByEmail(it.email) ?: throw IllegalArgumentException("이메일이 존재하지 않습니다")
```
처럼 수정하는 건 어떨까요?

이유가 있어서 이것만 `!!` 처리를 하는건지 조금 궁금하네요!ㅎㅎㅎ